src/transport/smtp/transport.rs

```diff
@@ -138,13 +150,6 @@ impl SmtpTransportBuilder {
         SmtpClient { info: self.info }
     }
 
-    /// Build the transport with custom pool settings
```

Comment:
Does that mean users cannot modify pool's settings?

Reply:
Right. I guess what we could do is have those options be configurable directly through `SmtpTransportBuilder`, a bit like reqwest does https://docs.rs/reqwest/0.10.8/reqwest/struct.ClientBuilder.html#method.pool_idle_timeout

This way we won't be exposing r2d2. This is even more important when we implement connection pooling for the async version, since crates like `deadpool` get more frequent breaking changes.

Reply:
Ok, let's do it this way for now!